src/webots/sound/WbSoundEngine.cpp

```diff
@@ -128,20 +127,24 @@ bool WbSoundEngine::openAL() {
 void WbSoundEngine::setWorld(WbWorld *world) {
   if (world) {
     gWorld = world;
-    gViewpoint = world->viewpoint();
-    if (gViewpoint)
-      QObject::connect(gViewpoint, &WbViewpoint::cameraParametersChanged, &WbSoundEngine::updateListener);
+    if (gWorld && gWorld->viewpoint())
```

Comment:
That should be sufficient.
```suggestion
    if (gWorld)
```

Reply:
Indeed, doesn't appear to be necessary